### ISSUE TITLE
Transporter gets stuck while uploading to testflight.

### ISSUE THREAD

**Issue by justin-pierce** [fastlane/fastlane#19852]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Gets stuck while uploading to testflight. The last line printed is `[Transporter]: DBG-X:   parameter NewPackageName = xxxxxxx.itmsp`. First encountered it with a new project, but also found for an older project that the same configuration was working previously and now also gets stuck, so I'm not sure what changed. Tried updating Fastlane but it didn't seem to help. Manually uploaded using testflight on that same machine and it worked fine. Tried restarting PC. 

Seems to match issue https://github.com/fastlane/fastlane/issues/19428 -- wish they provided clarity for why they closed it. 

##### Command executed
```
        upload_to_testflight(
            api_key: api_key,
            team_id: "xxxxxxxx"
        )
```

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>12:55:25 INFO [2022-01-25 13:55:25.74]: [32m----------------------------------[0m
12:55:25 INFO [2022-01-25 13:55:25.74]: [32m--- Step: upload_to_testflight ---[0m
12:55:25 INFO [2022-01-25 13:55:25.74]: [32m----------------------------------[0m
12:55:25 INFO [2022-01-25 13:55:25.74]: Creating authorization token for App Store Connect API
12:55:25 DEBUG [2022-01-25 13:55:25.78]: App identifier (xxxxxxxx)
12:55:29 INFO [2022-01-25 13:55:29.82]: [32mReady to upload new build to TestFlight (App: xxxxxxxx)...[0m
12:55:29 DEBUG [2022-01-25 13:55:29.83]: App Platform (ios)
12:55:35 INFO [2022-01-25 13:55:35.13]: [32mWrote XML data to '/var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/d20220125-31079-bnziro/xxxxxxxx-6f5f8380-f5f9-4b3c-9275-d1094e3971bc.itmsp'[0m
12:55:37 INFO [2022-01-25 13:55:37.96]: Going to upload updated app to App Store Connect
12:55:37 INFO [2022-01-25 13:55:37.96]: [32mThis might take a few minutes. Please don't interrupt the script.[0m
12:55:37 DEBUG [2022-01-25 13:55:37.96]: xcrun iTMSTransporter -m upload -jwt YourJWT -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000 2>&1
12:55:44 DEBUG [2022-01-25 13:55:44.55]: [Transporter]: INFO: Configuring logging...
12:55:44 DEBUG [2022-01-25 13:55:44.55]: [Transporter]: INFO: Logging level set to eXtreme
12:55:51 DEBUG [2022-01-25 13:55:51.03]: [Transporter]: DEBUG: SMART-CLIENT: initializing data center locking ops: [authenticateForSession, validateAssets]
12:55:51 DEBUG [2022-01-25 13:55:51.10]: [Transporter]: DEBUG: SMART-CLIENT: initializing data centers with: [contentdelivery01.itunes.apple.com, contentdelivery02.itunes.apple.com]
12:55:51 DEBUG [2022-01-25 13:55:51.10]: [Transporter]: INFO: iTMSTransporter : iTunes Store Transporter [2.3.0]
12:55:51 DEBUG [2022-01-25 13:55:51.10]: [Transporter]: INFO: OS identifier: Mac OS X 10.15.6 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
12:55:51 DEBUG [2022-01-25 13:55:51.10]: [Transporter]: INFO: Memory: [JVM] 998M free, 1024M total, 2048M max [System] (Physical) 2406M free, 8192M total (Swap) 928M free, 1024M total
12:55:51 DEBUG [2022-01-25 13:55:51.11]: [Transporter]: DBG-X: Memory: [JVM] 998M free, 1024M total, 2048M max [System] (Physical) 2406M free, 8192M total (Swap) 928M free, 1024M total
12:55:51 DEBUG [2022-01-25 13:55:51.11]: [Transporter]: DBG-X: Using operation named: authenticateForSession
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: DBG-X: Apple's web service operation input parameters:
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.6 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2022-01-25T13:55:51-06:00
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: DBG-X:   parameter StatisticsPreviousCallDurationInSecs = 0.280538405
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: DBG-X:   parameter Version = 2.3.0
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
12:55:51 DEBUG [2022-01-25 13:55:51.14]: [Transporter]: INFO: id = 20220125135551-708
12:55:51 DEBUG [2022-01-25 13:55:51.15]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); current data center is null
12:55:51 DEBUG [2022-01-25 13:55:51.15]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); iterator was null, setting it
12:55:51 DEBUG [2022-01-25 13:55:51.16]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); setting current data center to: contentdelivery01.itunes.apple.com
12:55:51 DEBUG [2022-01-25 13:55:51.16]: [Transporter]: INFO: iTMSTransporter Correlation Key: 2257bf5c-4703-4527-90b1-0ca497da8651-0001
12:55:51 DEBUG [2022-01-25 13:55:51.17]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DEBUG: SMART-CLIENT: pinning data center to: contentdelivery01.itunes.apple.com
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X: Apple's web service operation return value:
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X:   parameter SharedSecret = **hidden value**
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X:   parameter MultipartUploadsEnabled = true
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X:   parameter TxHeaders = {jenga=true}
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X:   parameter SessionExpiration = 2022-01-29T19:55:51.578Z
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X:   parameter SessionId = CNmrDBIQDP340IrEQyCKTRQYP8nKFA==
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
12:55:51 DEBUG [2022-01-25 13:55:51.80]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X: Using operation named: lookupSoftwareForBundleId
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X: Apple's web service operation input parameters:
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X:   parameter BundleId = xxxxxxxx
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.6 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X:   parameter SoftwareTypeEnum = ios
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2022-01-25T13:56:19-06:00
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000
12:56:19 DEBUG [2022-01-25 13:56:19.68]: [Transporter]: DBG-X:   parameter Version = 2.3.0
12:56:19 DEBUG [2022-01-25 13:56:19.69]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
12:56:19 DEBUG [2022-01-25 13:56:19.69]: [Transporter]: INFO: id = 20220125135619-675
12:56:19 DEBUG [2022-01-25 13:56:19.69]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
12:56:19 DEBUG [2022-01-25 13:56:19.69]: [Transporter]: INFO: iTMSTransporter Correlation Key: 2257bf5c-4703-4527-90b1-0ca497da8651-0001
12:56:19 DEBUG [2022-01-25 13:56:19.69]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X: Apple's web service operation return value:
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter ProviderPublicId = 44fd3952-b3a5-4542-b0e4-d1b3569fcfd6
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter DisplayKeys = [Application, Version Number, SKU Number, Primary Language, Copyright, Type, Apple ID]
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter Applications = {xxxxxxxx 1.0 (iOS App)=xxxxxxxx}
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter ValidIconSizes = [57x57]
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter SessionExpiration = 2022-01-29T19:56:19.941Z
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter Attributes = [{Apple ID=xxxxxxxx, AppleID=xxxxxxxx, WWDRIdentifier=xxxxxxxx, ExistingBundleIdentifier=xxxxxxxx, SKU Number=HOLOX, Version Number=2, Copyright=2021 NexTech AR Solutions, IconURL=https://a4.mzstatic.com/us/r30/Purple116/v4/ea/60/c5/ea60c5ab-c939-cd4f-98ee-ff7ccd77dc72/AppIcon-1x_U007emarketing-0-7-0-85-220.png?downloadKey=1643745380_e2f9e9024f098d39eb6c61663ee690e1, Type=iOS App, Primary Language=en-US, ShortVersion=1.0, ReservedBundleIdentifier=xxxxxxxx, DidOptInToMacAppStore=true, SoftwareTypeEnum=Purple, Application=xxxxxxxx, MarketingVersion=1.0}]
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter Success = true
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter PerformZipFilenameVerification = true
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter ProviderName = xxxxxxxx
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter SkipValidateSupportedArchitectures = true
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter PerformIconVerification = true
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter PerformCodesignVerification = anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.1] exists and certificate leaf[field.1.2.840.113635.100.6.1.4] exists
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter PerformIconSizeVerification = true
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter DisallowedPaths = [SC_Info]
12:56:20 DEBUG [2022-01-25 13:56:20.25]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
12:56:22 DEBUG [2022-01-25 13:56:22.96]: [Transporter]: INFO: Upload mode selected.
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X: Memory: [JVM] 952M free, 1024M total, 2048M max [System] (Physical) 2212M free, 8192M total (Swap) 928M free, 1024M total
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X: Using operation named: authenticateForTransportDiscovery
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X: Apple's web service operation input parameters:
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.6 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2022-01-25T13:56:22-06:00
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X:   parameter Version = 2.3.0
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: INFO: id = 20220125135622-877
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: INFO: iTMSTransporter Correlation Key: 2257bf5c-4703-4527-90b1-0ca497da8651-0001
12:56:22 DEBUG [2022-01-25 13:56:22.98]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
12:56:24 DEBUG [2022-01-25 13:56:24.31]: [Transporter]: DBG-X: Apple's web service operation return value:
12:56:24 DEBUG [2022-01-25 13:56:24.31]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
12:56:24 DEBUG [2022-01-25 13:56:24.31]: [Transporter]: DBG-X:   parameter PreferredTransports = [HTTP, Signiant, Aspera]
12:56:24 DEBUG [2022-01-25 13:56:24.31]: [Transporter]: DBG-X:   parameter TransportHosts.Signiant = [sgr601.apple.com:44001, sgr602.apple.com:44001, sgr603.apple.com:44001, sgr604.apple.com:44001]
12:56:24 DEBUG [2022-01-25 13:56:24.31]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
12:56:24 DEBUG [2022-01-25 13:56:24.31]: [Transporter]: DBG-X:   parameter SessionExpiration = 2022-01-29T19:56:22.419Z
12:56:24 DEBUG [2022-01-25 13:56:24.31]: [Transporter]: DBG-X:   parameter TransportHosts.Aspera = [vgr502.apple.com:33001, vgr504.apple.com:33001, vgr503.apple.com:33001, vgr501.apple.com:33001]
12:56:24 DEBUG [2022-01-25 13:56:24.31]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
12:56:24 DEBUG [2022-01-25 13:56:24.32]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
12:56:24 DEBUG [2022-01-25 13:56:24.32]: [Transporter]: INFO: Transporter's command line arguments are: -m upload -jwt **hidden value** -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000
12:56:24 DEBUG [2022-01-25 13:56:24.32]: [Transporter]: INFO: Gathering the list of valid files from the package ...
12:56:24 DEBUG [2022-01-25 13:56:24.32]: [Transporter]: DEBUG:   metadata.xml will be verified by Apple's web service to determine if it is a valid file.
12:56:24 DEBUG [2022-01-25 13:56:24.32]: [Transporter]: DEBUG:   e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa will be verified by Apple's web service to determine if it is a valid file.
12:56:24 DEBUG [2022-01-25 13:56:24.32]: [Transporter]: INFO: Finished gathering the list of valid files from the package.
12:56:24 DEBUG [2022-01-25 13:56:24.33]: [Transporter]: INFO: Examining the package at: /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/DeveloperAPIUpload7238634323382259322/xxxxxxxx.itmsp
12:56:24 DEBUG [2022-01-25 13:56:24.33]: [Transporter]: INFO: Ensuring that package has well formed metadata file...
12:56:24 DEBUG [2022-01-25 13:56:24.33]: [Transporter]: DEBUG: Getting case-sensitive metadata xml filename...
12:56:24 DEBUG [2022-01-25 13:56:24.33]: [Transporter]: DEBUG:   Case-sensitive metadata filename is: metadata.xml
12:56:24 DEBUG [2022-01-25 13:56:24.33]: [Transporter]: INFO: Performing authentication of package xxxxxxxx.itmsp ...
12:56:24 DEBUG [2022-01-25 13:56:24.39]: [Transporter]: DBG-X: Memory: [JVM] 940M free, 1024M total, 2048M max [System] (Physical) 2201M free, 8192M total (Swap) 928M free, 1024M total
12:56:24 DEBUG [2022-01-25 13:56:24.39]: [Transporter]: DBG-X: Deflated 634 bytes to 476
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X: Using operation named: validateMetadata
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X: Apple's web service operation input parameters:
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter Files = [metadata.xml, e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa]
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter Metadata = <?xml version="1.0" encoding="UTF-8"?>
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter MetadataChecksum = 1fce74b7298cea03104e241fd97b7884
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter MetadataCompressed = (suppressed)
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter MetadataInfo = {app_platform=ios, primary_bundle_identifier=, device_id=, bundle_identifier=xxxxxxxx, packageVersion=software5.4, apple_id=xxxxxxxx, asset_types=[bundle], bundle_version=144, bundle_short_version_string=1.0.1}
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.6 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter PackageName = xxxxxxxx.itmsp
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter PackageSize = 153871022
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2022-01-25T13:56:24-06:00
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter Throttle = 100000
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter Version = 2.3.0
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: INFO: id = 20220125135624-904
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: INFO: iTMSTransporter Correlation Key: 2257bf5c-4703-4527-90b1-0ca497da8651-0001
12:56:24 DEBUG [2022-01-25 13:56:24.40]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X: Apple's web service operation return value:
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter TxHeaders = {jenga=true}
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter SessionExpiration = 2022-01-29T19:56:24.496Z
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter MinimumFileSizeThresholdForCheckum = 104857600
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionFormat = binary
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionThresholds = {in-app-purchase-content={extensions=[pkg], size=100000000}, ccats={extensions=[pdf], size=100000000}, product-archive={extensions=[pkg], size=100000000}, developer-id-package={extensions=[zip, pkg, dmg], size=100000000}, bundle={extensions=[zip, ipa], size=100000000}, large-icon={extensions=[jpg, png, jpeg], size=100000000}}
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter Success = true
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter FilesToUpload = [metadata.xml, e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa]
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter AssetsToDescribe = [{Role=source, Filename=e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa, ShouldDescribeAsset=true, UTI=com.apple.ipa}]
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter CPUToken = 1fce4670-51d3-421d-9ff5-e7689aad7fc4/1643140586651
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter Checksums = {e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa=e89140934b7d7c16fb22bf2248239a39}
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter FileSizes = {e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa=153870388}
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: DBG-X:   parameter ProviderNameForToken = xxxxxxxx
12:56:27 DEBUG [2022-01-25 13:56:27.15]: [Transporter]: INFO: The list of files requested for upload by Apple is: [metadata.xml, e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa].  These are the files that will be uploaded.
12:56:27 DEBUG [2022-01-25 13:56:27.16]: [Transporter]: INFO: Starting media analysis of assets
12:56:27 DEBUG [2022-01-25 13:56:27.54]: [Transporter]: DEBUG: Handling asset: filename=e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa,uti=com.apple.ipa,role=source,describe=true
12:56:27 DEBUG [2022-01-25 13:56:27.54]: [Transporter]: DEBUG: Using container type tool name and version: iOS App Describer:1.26
12:56:27 DEBUG [2022-01-25 13:56:27.54]: [Transporter]: DEBUG: Using path: /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/DeveloperAPIUpload7238634323382259322/xxxxxxxx.itmsp/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa
12:56:27 DEBUG [2022-01-25 13:56:27.55]: [Transporter]: DEBUG: Using the provided assetDescription: /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa10616087773346338941swinfo.plist
12:56:27 DEBUG [2022-01-25 13:56:27.59]: [Transporter]: DBG-X: description length:0
12:56:27 DEBUG [2022-01-25 13:56:27.59]: [Transporter]: DBG-X: Asset description: 
12:56:27 DEBUG [2022-01-25 13:56:27.59]: [Transporter]: DEBUG: Asset successfully described: com.apple.jingle.leghorn.software.AbstractSoftwareDescriber$1@3d3c886f
12:56:27 DEBUG [2022-01-25 13:56:27.74]: [Transporter]: DBG-X: Deflated 648 bytes to 385
12:56:27 DEBUG [2022-01-25 13:56:27.74]: [Transporter]: INFO: Asset media analysis has completed
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X: Memory: [JVM] 917M free, 1024M total, 2048M max [System] (Physical) 2178M free, 8192M total (Swap) 928M free, 1024M total
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X: Using operation named: lookupTransportDiagnostic
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X: Apple's web service operation input parameters:
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.6 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2022-01-25T13:56:27-06:00
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X:   parameter TxTransportVersions = {SigniantTCP=1.2.0, Aspera=1.2.1, Signiant=1.2.0, HTTP=1.4.3}
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X:   parameter Version = 2.3.0
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: INFO: id = 20220125135627-842
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: INFO: iTMSTransporter Correlation Key: 2257bf5c-4703-4527-90b1-0ca497da8651-0001
12:56:27 DEBUG [2022-01-25 13:56:27.75]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
12:56:29 DEBUG [2022-01-25 13:56:29.82]: [Transporter]: DBG-X: Apple's web service operation return value:
12:56:29 DEBUG [2022-01-25 13:56:29.82]: [Transporter]: DBG-X:   parameter AsperaLoggingLevel = OFF
12:56:29 DEBUG [2022-01-25 13:56:29.82]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
12:56:29 DEBUG [2022-01-25 13:56:29.82]: [Transporter]: DBG-X:   parameter SigniantLoggingLevel = OFF
12:56:29 DEBUG [2022-01-25 13:56:29.82]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
12:56:29 DEBUG [2022-01-25 13:56:29.82]: [Transporter]: DBG-X:   parameter SessionExpiration = 2022-01-29T19:56:27.851Z
12:56:29 DEBUG [2022-01-25 13:56:29.82]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
12:56:29 DEBUG [2022-01-25 13:56:29.82]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
12:56:29 DEBUG [2022-01-25 13:56:29.85]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
12:56:29 DEBUG [2022-01-25 13:56:29.85]: [Transporter]: DBG-X: Using operation named: createTestReservation
12:56:29 DEBUG [2022-01-25 13:56:29.85]: [Transporter]: DBG-X: Apple's web service operation input parameters:
12:56:29 DEBUG [2022-01-25 13:56:29.86]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
12:56:29 DEBUG [2022-01-25 13:56:29.86]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
12:56:29 DEBUG [2022-01-25 13:56:29.86]: [Transporter]: DBG-X:   parameter fileDescriptions = ( one FileDescription )
12:56:29 DEBUG [2022-01-25 13:56:29.86]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
12:56:29 DEBUG [2022-01-25 13:56:29.86]: [Transporter]: DBG-X:   parameter NewPackageName = http-transport-diagnostic-20220125195629-848-3427.itmsp
12:56:29 DEBUG [2022-01-25 13:56:29.86]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.6 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
12:56:29 DEBUG [2022-01-25 13:56:29.86]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000
12:56:29 DEBUG [2022-01-25 13:56:29.86]: [Transporter]: DBG-X:   parameter Version = 2.3.0
12:56:29 DEBUG [2022-01-25 13:56:29.86]: [Transporter]: INFO: id = 20220125135629-328
12:56:29 DEBUG [2022-01-25 13:56:29.99]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
12:56:29 DEBUG [2022-01-25 13:56:29.99]: [Transporter]: INFO: iTMSTransporter Correlation Key: 2257bf5c-4703-4527-90b1-0ca497da8651-0001
12:56:30 DEBUG [2022-01-25 13:56:29.99]: [Transporter]: DEBUG: DataCenter: contentdelivery01.itunes.apple.com
12:56:30 DEBUG [2022-01-25 13:56:30.72]: [Transporter]: DBG-X: Apple's web service operation return value:
12:56:30 DEBUG [2022-01-25 13:56:30.72]: [Transporter]: DBG-X:   parameter Reservations = ( one Reservation )
12:56:30 DEBUG [2022-01-25 13:56:30.72]: [Transporter]: DBG-X:   parameter SessionExpiration = 2022-01-29T19:56:29.585Z
12:56:30 DEBUG [2022-01-25 13:56:30.72]: [Transporter]: DBG-X:   parameter Success = true
12:56:30 DEBUG [2022-01-25 13:56:30.73]: [Transporter]: INFO: Setting transport log file: /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/http-transport-diagnostic-20220125195629-848-3427.itmsp13317985268414990260.log
12:56:30 DEBUG [2022-01-25 13:56:30.73]: [Transporter]: INFO: Starting upload for packageName: http-transport-diagnostic-20220125195629-848-3427.itmsp
12:56:30 DEBUG [2022-01-25 13:56:30.73]: [Transporter]: INFO: Thread Pool Configuration: {rateLimit=100000 KBs, minConnections=6, coreThreads=6, perCoreThreadMultiplier=1.65, processors=4, maxThreads=7, maxConnections=6}
12:56:30 DEBUG [2022-01-25 13:56:30.74]: [Transporter]: INFO: Reservation CPUSource115/v4/54/75/ab/5475ab98-3b4a-9022-e36e-e8612711d40c/http-transport-diagnostic-13688048461132261734.txt created for file: (http-transport-diagnostic-13688048461132261734.txt), part count:(1), file size: (200)
12:56:30 DEBUG [2022-01-25 13:56:30.75]: [Transporter]: INFO: Computing total size of files to upload ...
12:56:30 DEBUG [2022-01-25 13:56:30.75]: [Transporter]: DBG-X:   adding file for % complete logging: http-transport-diagnostic-13688048461132261734.txt , size: 200
12:56:30 DEBUG [2022-01-25 13:56:30.75]: [Transporter]: DEBUG: Setting total size of files for upload to: 200
12:56:30 DEBUG [2022-01-25 13:56:30.75]: [Transporter]: INFO: Done computing total size of files to upload.
12:56:30 DEBUG [2022-01-25 13:56:30.76]: [Transporter]: INFO: Starting upload for reservation: (CPUSource115/v4/54/75/ab/5475ab98-3b4a-9022-e36e-e8612711d40c/http-transport-diagnostic-13688048461132261734.txt) file: (http-transport-diagnostic-13688048461132261734.txt/1) size: 200
12:56:31 DEBUG [2022-01-25 13:56:31.08]: [Transporter]: INFO:      File: http-transport-diagnostic-13688048461132261734.txt 200/200, 100.00% completed
12:56:31 DEBUG [2022-01-25 13:56:31.08]: [Transporter]: INFO: Package upload progress: 100.00% completed
12:56:31 DEBUG [2022-01-25 13:56:31.18]: [Transporter]: INFO: Finished part upload for: (http-transport-diagnostic-13688048461132261734.txt/1) 1.85 KB/s in 0 seconds
12:56:31 DEBUG [2022-01-25 13:56:31.18]: [Transporter]: INFO: Finished upload for reservation: (CPUSource115/v4/54/75/ab/5475ab98-3b4a-9022-e36e-e8612711d40c/http-transport-diagnostic-13688048461132261734.txt) file: (http-transport-diagnostic-13688048461132261734.txt)
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X: Using operation named: commitTestReservation
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X: Apple's web service operation input parameters:
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: INFO: Waiting on reservation commit(s) to finish.
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X:   parameter NewPackageName = http-transport-diagnostic-20220125195629-848-3427.itmsp
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.6 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X:   parameter reservations = [CPUSource115/v4/54/75/ab/5475ab98-3b4a-9022-e36e-e8612711d40c/http-transport-diagnostic-13688048461132261734.txt]
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DBG-X:   parameter Version = 2.3.0
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: INFO: id = 20220125135631-825
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: INFO: iTMSTransporter Correlation Key: 2257bf5c-4703-4527-90b1-0ca497da8651-0001
12:56:31 DEBUG [2022-01-25 13:56:31.19]: [Transporter]: DEBUG: DataCenter: contentdelivery01.itunes.apple.com
12:56:31 DEBUG [2022-01-25 13:56:31.56]: [Transporter]: DBG-X: Apple's web service operation return value:
12:56:31 DEBUG [2022-01-25 13:56:31.56]: [Transporter]: DBG-X:   parameter SessionExpiration = 2022-01-29T19:56:31.295Z
12:56:31 DEBUG [2022-01-25 13:56:31.56]: [Transporter]: DBG-X:   parameter Success = true
12:56:31 DEBUG [2022-01-25 13:56:31.56]: [Transporter]: INFO: Reservation committed: (CPUSource115/v4/54/75/ab/5475ab98-3b4a-9022-e36e-e8612711d40c/http-transport-diagnostic-13688048461132261734.txt) file: (http-transport-diagnostic-13688048461132261734.txt)
12:56:31 DEBUG [2022-01-25 13:56:31.57]: [Transporter]: INFO: All reservation commit(s) finished.
12:56:31 DEBUG [2022-01-25 13:56:31.57]: [Transporter]: INFO: 241.84 B/s in 0 seconds
12:56:31 DEBUG [2022-01-25 13:56:31.58]: [Transporter]: DEBUG: Using token file path /Users/xxxxxxxx/Library/Caches/com.apple.amp.itmstransporter/UploadTokens/xxxxxxxx.local_xxxxxxxx_xxxxxxxx.itmsp.token
12:56:31 DEBUG [2022-01-25 13:56:31.58]: [Transporter]: DEBUG: Attempting exclusive file lock on token file
12:56:31 DEBUG [2022-01-25 13:56:31.58]: [Transporter]: DEBUG: Token exists, examining
12:56:31 DEBUG [2022-01-25 13:56:31.83]: [Transporter]: DEBUG: Token pid: 25479, found process = false; my pid: 37418
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DEBUG: Reusing existing token: 9501f683-d1ea-4158-a8c9-18194044b135/1643131156288
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DEBUG: Set token: 9501f683-d1ea-4158-a8c9-18194044b135/1643131156288
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X: Memory: [JVM] 894M free, 1024M total, 2048M max [System] (Physical) 2170M free, 8192M total (Swap) 928M free, 1024M total
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X: Deflated 634 bytes to 476
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X: Using operation named: validateAssets
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X: Apple's web service operation input parameters:
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X:   parameter CPUToken = 9501f683-d1ea-4158-a8c9-18194044b135/1643131156288
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X:   parameter Files = [metadata.xml, e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa]
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X:   parameter MetadataChecksum = 1fce74b7298cea03104e241fd97b7884
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X:   parameter MetadataCompressed = (suppressed)
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X:   parameter MetadataInfo = {app_platform=ios, primary_bundle_identifier=, device_id=, bundle_identifier=xxxxxxxx, packageVersion=software5.4, apple_id=xxxxxxxx, asset_types=[bundle], bundle_version=144, bundle_short_version_string=1.0.1}
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 10.15.6 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
12:56:31 DEBUG [2022-01-25 13:56:31.86]: [Transporter]: DBG-X:   parameter PackageName = xxxxxxxx.itmsp
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter PackageSize = 153871022
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter RequestUUID = 0c0d4bc0-3174-4f18-942c-a1f914d3e8e0
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2022-01-25T13:56:31-06:00
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter StreamingInfoList = [file: e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa10616087773346338941swinfo.plist9895066135377200136.swinfo.gz, content-length: 227858, content-type: application/binary, MD5: 7aa065264d443534a5fdbcf888aa9a1b]
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter Throttle = 100000
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter Transport = HTTP
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -jwt **hidden value** -assetFile /var/folders/8v/9s_vtlwj1vz0ny0mjqdrksjr0000gn/T/e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa -k 100000
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter TxTransportVersions = {SigniantTCP=1.2.0, Aspera=1.2.1, Signiant=1.2.0, HTTP=1.4.3}
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter Version = 2.3.0
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: INFO: id = 20220125135631-328
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: INFO: iTMSTransporter Correlation Key: 2257bf5c-4703-4527-90b1-0ca497da8651-0001
12:56:31 DEBUG [2022-01-25 13:56:31.87]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
12:56:41 DEBUG [2022-01-25 13:56:41.56]: [Transporter]: DBG-X: Apple's web service operation return value:
12:56:41 DEBUG [2022-01-25 13:56:41.56]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
12:56:41 DEBUG [2022-01-25 13:56:41.56]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
12:56:41 DEBUG [2022-01-25 13:56:41.56]: [Transporter]: DBG-X:   parameter SessionExpiration = 2022-01-29T19:56:32.246Z
12:56:41 DEBUG [2022-01-25 13:56:41.56]: [Transporter]: DBG-X:   parameter MinimumFileSizeThresholdForCheckum = 104857600
12:56:41 DEBUG [2022-01-25 13:56:41.56]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionFormat = binary
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionThresholds = {in-app-purchase-content={extensions=[pkg], size=100000000}, ccats={extensions=[pdf], size=100000000}, product-archive={extensions=[pkg], size=100000000}, developer-id-package={extensions=[zip, pkg, dmg], size=100000000}, bundle={extensions=[zip, ipa], size=100000000}, large-icon={extensions=[jpg, png, jpeg], size=100000000}}
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter ResumeFlag = false
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter Success = true
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter FilesToUpload = [metadata.xml, e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa]
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter AssetsToDescribe = [{Role=source, Filename=e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa, ShouldDescribeAsset=true, UTI=com.apple.ipa}]
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter CPUToken = c44ef4e6-abbe-47f9-825f-9a85f05e36ee/1643140599851
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter Checksums = {e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa=e89140934b7d7c16fb22bf2248239a39}
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter SigniantLoggingLevel = INFO
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter FileSizes = {e052be79-b3b0-43d3-b4e3-96642c62f3aa.ipa=153870388}
12:56:41 DEBUG [2022-01-25 13:56:41.57]: [Transporter]: DBG-X:   parameter NewPackageName = xxxxxxxx-20220125115640-228.itmsp</pre>
</details>

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                   |
| --------------------------- | ------------------------------------------------------- |
| OS                          | 10.15.6                                                 |
| Ruby                        | 3.0.3                                                   |
| Bundler?                    | false                                                   |
| Git                         | git version 2.24.3 (Apple Git-128)                      |
| Installation Source         | /usr/local/Cellar/fastlane/2.201.2/libexec/bin/fastlane |
| Host                        | Mac OS X 10.15.6 (19G2021)                              |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/3.0.3/lib                        |
| OpenSSL Version             | OpenSSL 1.1.1l  24 Aug 2021                             |
| Is contained                | false                                                   |
| Is homebrew                 | true                                                    |
| Is installed via Fabric.app | false                                                   |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/             |
| Xcode Version               | 12.4                                                    |
| Swift Version               | 5.3.2                                                   |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.201.2 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| did_you_mean                    | 1.5.0        |
| atomos                          | 0.1.3        |
| rexml                           | 3.2.5        |
| CFPropertyList                  | 3.0.5        |
| claide                          | 1.1.0        |
| colored2                        | 3.1.2        |
| nanaimo                         | 0.3.0        |
| xcodeproj                       | 1.21.0       |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 1.8.0        |
| terminal-table                  | 1.8.0        |
| plist                           | 3.6.0        |
| public_suffix                   | 4.0.6        |
| addressable                     | 2.8.0        |
| multipart-post                  | 2.0.0        |
| word_wrap                       | 1.0.0        |
| optparse                        | 0.1.1        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| artifactory                     | 3.0.15       |
| babosa                          | 1.0.4        |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| excon                           | 0.90.0       |
| faraday-em_http                 | 1.0.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-excon                   | 1.1.0        |
| faraday-httpclient              | 1.0.1        |
| faraday-multipart               | 1.0.3        |
| faraday-net_http                | 1.0.1        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-patron                  | 1.0.0        |
| faraday-rack                    | 1.0.0        |
| faraday-retry                   | 1.0.3        |
| ruby2_keywords                  | 0.0.5        |
| faraday                         | 1.9.3        |
| unf_ext                         | 0.0.8        |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| http-cookie                     | 1.0.4        |
| faraday-cookie_jar              | 0.0.7        |
| faraday_middleware              | 1.2.0        |
| fastimage                       | 2.2.6        |
| gh_inspector                    | 1.1.3        |
| json                            | 2.5.1        |
| mini_magick                     | 4.11.0       |
| naturally                       | 2.2.1        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.3        |
| xcpretty-travis-formatter       | 1.0.1        |
| dotenv                          | 2.7.6        |
| bundler                         | 2.2.32       |
| simctl                          | 1.6.8        |
| jwt                             | 2.3.0        |
| uber                            | 0.1.0        |
| declarative                     | 0.0.20       |
| trailblazer-option              | 0.1.2        |
| representable                   | 3.1.1        |
| retriable                       | 3.1.2        |
| mini_mime                       | 1.1.2        |
| memoist                         | 0.16.2       |
| multi_json                      | 1.15.0       |
| os                              | 1.1.4        |
| signet                          | 0.16.0       |
| googleauth                      | 1.1.0        |
| httpclient                      | 2.8.3        |
| webrick                         | 1.7.0        |
| google-apis-core                | 0.4.2        |
| google-apis-playcustomapp_v1    | 0.7.0        |
| google-apis-androidpublisher_v3 | 0.15.0       |
| google-cloud-env                | 1.5.0        |
| google-cloud-errors             | 1.2.0        |
| google-cloud-core               | 1.6.0        |
| google-apis-iamcredentials_v1   | 0.10.0       |
| google-apis-storage_v1          | 0.11.0       |
| rake                            | 13.0.6       |
| digest-crc                      | 0.6.4        |
| google-cloud-storage            | 1.36.0       |
| emoji_regex                     | 3.2.3        |
| jmespath                        | 1.5.0        |
| aws-partitions                  | 1.549.0      |
| aws-eventstream                 | 1.2.0        |
| aws-sigv4                       | 1.4.0        |
| aws-sdk-core                    | 3.125.5      |
| aws-sdk-kms                     | 1.53.0       |
| aws-sdk-s3                      | 1.111.3      |
| set                             | 1.0.1        |
| forwardable                     | 1.3.2        |
| logger                          | 1.4.3        |
| pathname                        | 0.1.0        |
| shellwords                      | 0.1.0        |
| cgi                             | 0.2.1        |
| date                            | 3.1.3        |
| timeout                         | 0.1.1        |
| stringio                        | 3.0.1        |
| securerandom                    | 0.1.0        |
| uri                             | 0.10.1       |
| ipaddr                          | 1.2.2        |
| openssl                         | 2.2.1        |
| digest                          | 3.0.0        |
| io-nonblock                     | 0.1.0        |
| io-wait                         | 0.2.0        |
| zlib                            | 2.0.0        |
| resolv                          | 0.2.1        |
| time                            | 0.1.0        |
| open-uri                        | 0.1.0        |
| mutex_m                         | 0.1.1        |
| net-protocol                    | 0.1.1        |
| ostruct                         | 0.3.1        |
| english                         | 0.7.1        |
| erb                             | 2.2.0        |
| strscan                         | 3.0.1        |
| abbrev                          | 0.1.0        |
| io-console                      | 0.5.7        |
| tempfile                        | 0.1.1        |
| delegate                        | 0.2.0        |
| fileutils                       | 1.5.0        |
| tmpdir                          | 0.1.2        |
| base64                          | 0.1.0        |
| singleton                       | 0.1.1        |
| net-http                        | 0.1.1        |
| open3                           | 0.1.1        |
| nkf                             | 0.1.0        |
| prettyprint                     | 0.1.1        |
| pp                              | 0.2.1        |
| find                            | 0.1.0        |
| yaml                            | 0.1.1        |
| psych                           | 3.3.2        |
</details>

*generated on:* **2022-01-25**
</details>

<details>
  <pre> Runs from a Fastfile executed by Jenkins (with verbose turned on) running on a mac mini) </pre>
</details>

edit: also tried `DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS="-t DAV" fastlane deliver` but DAV appears to be no longer supported?

edit2: as per Apple docs, I also tried http as that seems to be the new firewall-free option. Got stuck in same place.

**Comment by justin-pierce** [comment]
I should note, if I try to upload a build with build version/number lower than last, it fails and exits correctly. It only hangs indefinitely when uploading a valid build.

**Comment by justin-pierce** [comment]
Setting env variables `LANG` and `LC_ALL` to `en_US.UTF-8` seems to have fixed it.

**Comment by Renner11** [comment]
When i try this i got the same issue and wanted the solution but still finding it can you give me some more suggestions?

**Comment by mousedownmike** [comment]
I had issues similar to these after updating to Monterey on my M1 MacBook Air.  It was incredibly frustrating but it started to work after I ran the following:

```shell
xcode-select --install
```

I'm still seeing lots of timeouts when I run with verbose but the upload eventually succeeds.
